pkg/go/graph/graph_builder_test.go

```diff
@@ -190,6 +190,110 @@ rankdir=BT
 1 -> 3 [style=dashed];
 2 -> 1 [style=dashed];
 3 -> 2 [style=dashed];
+}`,
+		},
+		`tuple_to_userset_one_related_type`: {
+			model: `
+				model
+					schema 1.1
+				type user
+				type document
+					relations
+						define parent: [folder]
+						define viewer: admin from parent
+				type folder
+					relations
+						define admin: [user]`,
+			expectedOutput: `digraph {
+graph [
+rankdir=BT
+];
+
+// Node definitions.
+0 [label=document];
+1 [label="document#parent"];
+2 [label=folder];
+3 [label="document#viewer"];
+4 [label="folder#admin"];
+5 [label=user];
+
+// Edge definitions.
+2 -> 1 [label=direct];
+4 -> 3 [headlabel="(document#parent)"];
```

Comment:
Why make it a label, rather than actually connect the edges? `document#parent` is not an independent node when it comes to TTU.

Reply:
What edges would you connect?

Reply:
<img width="1205" alt="image" src="https://github.com/user-attachments/assets/fb569170-123d-48d3-bbd1-1b057b506566">